### DISCUSSION TITLE
Feedback on documentation

### DISCUSSION THREAD

**Discussion by andymac4182** [markdown-confluence/markdown-confluence#268] [General]
Would like to understand 
- What is working?
- What is missing?
- What can be improved?

**Comment by mrjcleaver** [comment]
1. I didn't see a link to this discussion group (e.g. from the plugin setup page, the web page, etc), so the group took some hunting to find.
2. It'd be useful to have a troubleshooting section on https://obsidian-confluence.com/  
3. The troubleshooting guide should include a section on what to do if you see this error: 
![image](https://github.com/markdown-confluence/markdown-confluence/assets/59202/47764fd8-abb6-4643-8ad9-5a56e5369fd3)

Thanks ;)
   Martin

**Reply by andymac4182** [reply]
Honestly have never seen that error. What caused it? 
Happy to accept a new page being added to https://github.com/markdown-confluence/docs-obsidian-confluence or https://github.com/markdown-confluence/docs-markdown-confluence whichever one you want to :)

**Comment by MikamiShuji** [comment]
Gonna have to agree on the lack of Troubleshooting steps for the tool - it's hard to determine whether or not the toolkit can be used for a particular Confluence version/server. Especially if you're trying to use something like the Obsidian plugin.

Case in point - I'm having quite a lot of trouble setting the plugin up for my company's instance of Confluence and consistently running into the "Page Not Found" placeholder. At which point I'm wondering:

* What kind of URL the tool is attempting to compose for the config that I provided?
* Is there any way to infer the type of server used (Cloud/On-Premise) based on the kind of URL I'm expecting?
* What steps user needs to take in order to obtain all the necessariy info for cofiguration (i.e. finding the page ID)?

All of these would be extremely helpful to have in order to improve the setup process. Because for now, it can be a bit of a "hit or miss".

![image](https://github.com/markdown-confluence/markdown-confluence/assets/44070490/c1eebc56-688a-4aea-9987-d48db7cf5134)

**Comment by codingeek54** [comment]
Does this support bitbucket/github/gitlab repos for publishing markdown to confluence with docker

**Comment by baaastijn** [comment]
hello, +1 for troubleshooting especially from Github Action. I was stucked, then went to the CLI, and got my answers (multiple pages with the same name for example).
